recipes/yojimbo/all/conanfile.py

```diff
@@ -0,0 +1,115 @@
+                    "destination": os.path.join(self._source_subfolder, submodule["destination"]),
+                    "strip_root": True
+                }
+
+                tools.get(**submodule_data)
+                submodule_source = os.path.join(self._source_subfolder, path)
+                tools.rmdir(submodule_source)
+
+    def build(self):
+
+        # Before building we need to make some edits to the premake file to build using conan dependencies rather than local/bundled
+
+        # Generate the list of dependency include and library paths as strings
+        include_path_str = ', '.join(f'"{p}"' for p in self.deps_cpp_info["libsodium"].include_paths + self.deps_cpp_info["mbedtls"].include_paths)
+        lib_path_str = ', '.join(f'"{p}"' for p in self.deps_cpp_info["libsodium"].lib_paths + self.deps_cpp_info["mbedtls"].lib_paths)
+
+        if self.settings.os == "Windows":
+        
+            # Replace Windows directory seperator
+            include_path_str = include_path_str.replace("\\", "/")
+            lib_path_str = lib_path_str.replace("\\", "/")
+            
+            # Edit the premake script to use conan rather than bundled dependencies
+            tools.replace_in_file("%s/premake5.lua" % self._source_subfolder, "includedirs { \".\", \"./windows\"", "includedirs { \".\", %s" % include_path_str, strict=True)
+            tools.replace_in_file("%s/premake5.lua" % self._source_subfolder, "libdirs { \"./windows\" }", "libdirs { %s }" % lib_path_str, strict=True)
+            
+            # Edit the premake script to change the name of libsodium
+            tools.replace_in_file("%s/premake5.lua" % self._source_subfolder, "\"sodium\"", "\"libsodium\"", strict=True)
+            
+        else:
+        
+        	# Edit the premake script to use  conan rather than local dependencies
+            tools.replace_in_file("%s/premake5.lua" % self._source_subfolder, "\"/usr/local/include\"", include_path_str, strict=True)
+            
+            
+        # Build using premake
+            
+        if self.settings.compiler == "Visual Studio":
+            generator = "vs" + {"16": "2019",
+                                "15": "2017",
+                                "14": "2015",
+                                "12": "2013",
+                                "11": "2012",
+                                "10": "2010",
+                                "9": "2008",
+                                "8": "2005"}.get(str(self.settings.compiler.version))
+        else:
+            generator = "gmake2"
+
+        with tools.chdir(self._source_subfolder):
+            self.run("premake5 %s" % generator)
+            
+            if self.settings.compiler == "Visual Studio":
+                msbuild = MSBuild(self)
+                msbuild.build("Yojimbo.sln")
+            else:
+                config = "Debug" if self.settings.build_type == "Debug" else "Release"
+                config = "_x64"
```

Comment:
I assume you meant this?

```suggestion
                config = "Debug" if self.settings.build_type == "Debug" else "Release"
                config += "_x64"
```